### DISCUSSION TITLE
Consulta sobre Maven y versiones de Java.

### DISCUSSION THREAD

**Discussion by gebonora** [dds-jv/foro#101] [Consultas de herramientas] [answered]
Buenas, una consulta sobre Maven y versiones de Java para el tp.

Según tengo entendido la versión de JDK a usar se especifica en el archivo pom.xml del proyecto.
- Este archivo en sí es parte del proyecto (va al repo)?
- Como consecuencia tendríamos que usar todos la misma versión de JDK?
- JDK 14 está bien o recomiendan usar JDK 8?

Gracias!

**Comment by gastonprieto** [comment]
Buenas!

En el pom.xml decis para que versión minimo compilar. La jdk que se va a usar se define por afuera, por la variable de entorno JAVA_HOME + PATH.

El archivo `pom.xml` si va en el repositorio, porque es el que describe el proyecto y el que luego va a servir para reconstruirlo.

Recomendamos usar la JDK 1.8 porque estamos seguro que todos los frameworks que usamos andan con ella.

**Comment by gebonora** [comment, marked as answer]
Gracias! Cierro el issue.
